client/me/purchases/manage-purchase/index.jsx

```diff
@@ -168,7 +168,12 @@ class ManagePurchase extends Component {
 			return null;
 		}
 
-		if ( ! isRenewable( purchase ) || isExpired( purchase ) || isExpiring( purchase ) || ! getSelectedSite( this.props ) ) {
+		if (
```

Comment:
Interesting, when you see the if broken out like this, you realise how unreadable it is. It should probably be in another function, but obviously not in this PR.